proto/greenfield/storage/query.proto

```diff
@@ -128,6 +128,20 @@ service Query {
     option (google.api.http).get = "/greenfield/storage/is_price_changed/{bucket_name}";
   }
 
+  // Queries whether some members are in the group.
+  rpc QueryGroupMembersExist(QueryGroupMembersExistRequest) returns (QueryGroupMembersExistResponse) {
+    option (google.api.http).get = "/greenfield/storage/group_members_exist/{group_id}/{members}";
+  }
+
+  // Queries whether some groups are exist.
+  rpc QueryGroupsExist(QueryGroupsExistRequest) returns (QueryGroupsExistResponse) {
+    option (google.api.http).get = "/greenfield/storage/groups_exist/{group_names}/{group_owners}";
```

Comment:
- Query multiple groups under one user at once
- Query multiple groups under multiple users at once

Shall we use a more generic ways to support?

Reply:
OK. What about nested list? `[groupName1, groupName2, ...]` and `[[group1_members1, ...], [group2_members1, ...], ...]

Reply:
To keep things simple, I think we'd better only support `Query multiple groups under one user at once`.